config/foreman.migrations/20200305172758_enable_puma.rb

```diff
@@ -0,0 +1,6 @@
+if answers['foreman'].is_a?(Hash)
+    mod_answers = answers['foreman']
+    if mod_answers['passenger'] && !(mod_answers['ipa_authentication'] || mod_answers['keycloak'])
```

Comment:
Could you explain why we are caring about ipa_authentication and keycloak here?

Reply:
So my thought was to capture https://github.com/theforeman/puppet-foreman/blob/1e10f6c43ba7ff8353b19b6b7d1a3dcf0b5310cd/manifests/init.pp#L351-L360 but you may be right that it's not needed. I was confusing Apache and Passenger.

Reply:
I did not test every permutation, but from my investigation and attempted testing I couldn't see why external auth wouldn't work with puma setup.

Reply:
So should we remove the external auth check and enable migration for all? and see how things to?

Reply:
Yes, I was confused with a non-apache install. That is still a problem but one we don't see that often.